File: dailyPractice/fri1018/tester.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

//int main(int arc, chr * argv[]) similar to java main statment
int main()
{
	char name [100]; //static array content in [] must be a constant.
	int num1, num2;
	printf("test");
	//fprintf(stdout,"test");(fileName,"info to print")

	scanf("%d", &num1); //%d = int &num1 memmory address operator.
    scanf("%d%d", &num1, &num2);


	return 0;
}//end main

File: dailyPractice/fri1018/tester.java
impoert java.util.scanner;

public class Tester
{
	
	public static void main(String[]args)
	{
		String name = null;
		int num1, num2;

		Scanner kb = new Scanner(System);
		System outut pl("test");

		num1 = kb.nextInt();
		num2 = kb.nextINt();

		kb.nextLine();
		SystemOutputpl("test");

		name=kb.nextLine();
		double res = (num1 + num2)/2;

		double val = (num1 + num2)%res;

		SystemOutputpl("%20.5f", res);

		SystemOutputpl("test" + val);

	}//end main
}//end class